### ISSUE TITLE
Help on running L_DMI

### ISSUE THREAD

**Issue by DiegoOrtego** [Newbeeer/L_DMI#2]
Hi!

I would like to run L-DMI in CIFAR-10 and CIFAR-100.  I would appreciated a bit of help to run it properly.
As far as I understood from the paper/code, you pre-train with cross-entropy and then you apply the DMI loss. Am I right? Also, how many epochs do I need to run with cross-entropy? Which learning rate should I use? In my experience, when training with cross-entropy a high learning rate is desirable to prevent (to some extent) fitting the label noise. Furthermore, when applying the DMI loss, I can train with 0.00001 learning rate as suggested in the paper, right?
Many thanks in advance!

Best,
Diego.

**Comment by Newbeeer** [comment]
Hi Diego,

Please refer to our README. You need to run the CE.py file first and then run the DMI.py file. The CE.py will automatically save the checkpoint and DMI.py will load the checkpoint at the beginning. Note that you should keep the command line args the same (e.g. noise_amount, seed) when running the two python files.

Best,
Yilun

**Comment by DiegoOrtego** [comment]
Hi Yilun,

I will do what you describe for the experiments! Many thanks!
Btw, we published at ICML a paper on dealing with label noise "Unsipervised label noise modeling and loss correction". Please find here the code in case you are interested: https://github.com/PaulAlbert31/LabelNoiseCorrection

Best regards,
Diego.

**Comment by Newbeeer** [comment]
Great! Exciting to follow your work.
